roles/sap_netweaver_preconfigure/tasks/sapnote/assert-3119751.yml

```diff
@@ -5,46 +5,88 @@
     msg: "SAP note {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '^3119751$') | first).number }}
           (version {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '^3119751$') | first).version }}): Linux Requirements for SAP Kernel 754 and for SAP Kernel 788 and higher"
 
-- name: Get info about the compat-sap-c++-10.so file
+# Note: This file is only included for RHEL 8, so no further when condition is required here.
+
+- name: Identify all 'compat-sap-c++-NUM.so' symlinks with NUM >= 10
+  ansible.builtin.find:
+    paths: '/opt/rh/SAP/lib64'
+    patterns: "compat-sap-c\\+\\+-1[0-9].so"
+    file_type: link
+    use_regex: true
+  register: __sap_netweaver_preconfigure_register_find_compat_sap_cpp
+
+# Note: The symlink compat-sap-c++-NUM.so with NUM >= 10 will be available if the role sap_general_preconfigure has been run before,
+# which is a requirement.
+- name: Assert that there is at least one symlink '/opt/rh/SAP/lib64/compat-sap-c++-NUM.so' with NUM >= 10
+  ansible.builtin.assert:
+    that: __sap_netweaver_preconfigure_register_find_compat_sap_cpp.matched > 0
+    fail_msg: "FAIL: There is no symlink '/opt/rh/SAP/lib64/compat-sap-c++-NUM.so' with NUM >= 10!"
+    success_msg: "FAIL: There is at least one symlink '/opt/rh/SAP/lib64/compat-sap-c++-NUM.so' with NUM >= 10!"
```

Comment:
Is the "FAIL:" in the _success_ message text intended or a copy-paste remainder?

Reply:
The latter, thank for catching it!